lib/utils/types.js

```diff
@@ -415,3 +416,13 @@ function isUnaryExpression(node) {
 function isVariableDeclarator(node) {
   return node !== undefined && node.type === 'VariableDeclarator';
 }
+
+/**
+ * Check whether or not a token is a Punctuator.
+ *
+ * @param {Object} token The token to check.
+ * @returns {boolean} Whether or not the token is a Punctuator.
+ */
+function isPunctuator(token) {
```

Comment:
Nit: I actually want to avoid adding trivial type-checking functions here in `types.js` and eliminate the existing ones at some point. These functions are not really that helpful and they will make it harder for us to add TypeScript type-checking: https://github.com/ember-cli/eslint-plugin-ember/issues/1613

Reply:
I see. I came across this file where all the type-checking functions are there and hence added the util method. I have removed it now. Thank you.